docs/prisma2-cli.md

```diff
@@ -36,30 +36,30 @@ The following environment variables can be provided:
 
 #### `prisma2 init`
 
-Sets up Prisma (i.e. Photon and/or Lift) via an interactive wizard.
+Sets up Prisma (i.e., Photon and/or Lift) via an interactive wizard. You can specify your database connection and/or create a new database to work with. For a list with currently supported database engines, see the [Lift documentation](./lift/use-only-lift.md).
 
 #### `prisma2 dev`
 
-Starts Prisma [development mode](./development-mode.md).
+Starts Prisma [development mode](./development-mode.md). This opens a web application to interact with your database and defined models. Note that `Prisma Studio` relies on your `prisma.schema` file (and hence the automatically generated Photon client) to provide a fully featured user-interface with CRUD (create / read / update / delete) functionality.
 
 #### `prisma2 generate`
 
-Invokes the generators specified in the Prisma schema file.
+Invokes all generators defined in the `prisma.schema` file. For example, this creates the Photon client to interact with the underlying database. Read more about Photon and its capabilities [here](./photon/use-only-photon.md).
 
 #### `prisma2 introspect`
 
-Introspects the database and generates a data model from it.
+Introspects the database and generates a data model from it. Basically, it analyzes your (already existing) database and automatically creates the `prisma.schema` file for you. This is useful, if you already have an existing application and want to start using Lift.
```

Comment:
>  `prisma.schema` file

Same point as before, let's spell it out as "Prisma schema file".

> and want to start using Lift.

this is not only for Lift but also for Photon, so I suggest to rephrase this to: "and want to start using the Prisma Framework"

Reply:
also, there's a bit more nuance to this command. the use case you're describing actually is for `prisma2 init`. the `introspect` command is typically used in Photon-only projects where migrations aren't performed through Lift but the Prisma schema needs to be updated. more info here: https://github.com/prisma/prisma2/blob/master/docs/introspection.md

Reply:
(hopefully) changed with the next commit